### PR TITLE
O3-104 Login ESM should have tests for config values

### PR DESCRIPTION
**Pull request by nanfuka**
## Requirements
- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.

#### For changes to apps
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.
- [ ] I have updated the [esm-framework mock](https://github.com/openmrs/openmrs-esm-core/blob/master/packages/framework/esm-framework/mock.tsx) to reflect any API changes I have made.

### CODE REVIEW

File: packages/apps/esm-login-app/src/login/login.component.test.tsx [hunk 1]
```diff
@@ -136,4 +136,22 @@ describe(`<Login />`, () => {
       expect(wrapper.history.location.pathname).toBe("/login/location")
     );
   });
+  it("respects the logo configuration", () => {
+    mockedUseConfig.mockReturnValue({
```
Comment (denniskigen) [inline packages/apps/esm-login-app/src/login/login.component.test.tsx:140]: I'd probably move the custom logo config into an object like: 

```js
const customLogoConfig = {
  src: "https://some-image-host.com/foo.png",
  alt: "Custom logo",
}
```

And then override the logo config in `mockReturnValue` call like so:

```js
mockedUseConfig.mockReturnValue({
  ...mockConfig,
  logo: customLogoConfig,
});
```

I'd also remove the wrapper and just render like:

```js
renderWithRouter(Login, {
  loginLocations: loginLocations,
  isLoginEnabled: true,
});
```

And then query using `screen`:

```js
const logo = screen.getByAltText(customLogoConfig.alt);
// assert that default SVG logo is not displayed
expect(screen.queryByTitle(/openmrs logo/i)).not.toBeInTheDocument();
expect(logo).toHaveAttribute("src", customLogoConfig.src);
expect(logo).toHaveAttribute("alt", customLogoConfig.alt);
```

**Review by denniskigen — COMMENTED** [review]
Nice work, @nanfuka! 

I've left some feedback outlining a slightly better approach to constructing your spec.

File: packages/apps/esm-login-app/src/login/login.component.test.tsx (hunk 1 above)
Comment (nanfuka) [reply packages/apps/esm-login-app/src/login/login.component.test.tsx:144]: thank for the feedback, I have made the changes

**Review by denniskigen — APPROVED** [review]
LGTM! Thanks, @nanfuka!

**Pull request merged**